skpro/distributions/halfnormal.py

```diff
@@ -0,0 +1,210 @@
+# copyright: skpro developers, BSD-3-Clause License (see LICENSE file)
+"""Half-Normal probability distribution."""
+
+__author__ = ["SaiRevanth25"]
+
+import numpy as np
+import pandas as pd
+from scipy.special import erf, erfinv
+
+from skpro.distributions.base import BaseDistribution
+
+
+class HalfNormal(BaseDistribution):
```

Comment:
you could use the scipy adapter, scipy has `halfnorm` - although that's not necessary, expressions are straightforward. Just point this out, not a change request

Reply:
i think using the scipy adatpter would be better in this case!!

Reply:
i think using the scipy adapter would be better in this case!!